dependencies/O2Dependencies.cmake

```diff
@@ -116,10 +116,6 @@ find_package(benchmark CONFIG NAMES benchmark googlebenchmark)
 set_package_properties(benchmark PROPERTIES TYPE OPTIONAL)
 find_package(OpenMP)
 set_package_properties(OpenMP PROPERTIES TYPE OPTIONAL)
-if (OpenMP_FOUND)
```

Comment:
this is still needed until we enforce OpenMP as system package. See https://github.com/alisw/alidist/pull/1922.